pymeasure/experiment/parameters.py

```diff
@@ -293,6 +293,7 @@ def __repr__(self):
 
 class ListParameter(Parameter):
     """ :class:`.Parameter` sub-class that stores the value as a list.
+    String representation of value must be unique.
```

Comment:
One minor question: would it be possible to enforce/error-check this? I think that could help users a lot. (If so, a small test that check whether an error is raised might also be nice)

(maybe change `value` to `values` or `choices`)

Reply:
@moritzj29 this is very useful fix for `ListParameter`. If you find time, could please address @CasperSchippers comment, so that we can proceed to merge?

Reply:
added as suggested, thanks for the input!